### ISSUE TITLE
Searchbar placeholder style is becoming distorted.

### ISSUE THREAD

**Issue by otabekoff** [vuejs/vitepress#909]
### Describe the bug

![image](https://user-images.githubusercontent.com/60849894/177046463-7e392214-9150-4c1c-b604-d783a76b9bfb.png)

Look this. How bad is this? Ctrl+K placeholder is becoming ugly.

When I see its code. It was an SVG code. When I refresh and then see there was no any SVG code.

### Reproduction

Changing some styles and doing fixes. 

### Expected behavior

![image](https://user-images.githubusercontent.com/60849894/177046606-9a560714-0e72-42c8-9a97-344c85adc354.png)

It should look like this.

### System Info

```shell
System:
    OS: Windows 10 10.0.22621 | Note: this is wrong. I use Windows 11.
    CPU: (12) x64 Intel(R) Core(TM) i5-10400F CPU @ 2.90GHz
    Memory: 7.50 GB / 15.90 GB
  Binaries:
    Node: 16.14.2 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.18 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 8.5.0 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: Spartan (44.22621.160.0), Chromium (103.0.1264.44)
    Internet Explorer: 11.0.22621.1

Note: I don't believe this `envinfo` tool. Some of the above information may be incorrect.
```

### Validations

- [X] Follow our [Code of Conduct](https://vuejs.org/about/coc.html)
- [X] Read the [docs](https://vitepress.vuejs.org).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

**Comment by brc-dd** [comment]
Yeah we are aware of this (from much before alpha release IG). We tried fixing it, but docsearch/js keeps modifying the DOM. If you have any fix, then PRs are welcome.

Also, version of Windows 11 is 10 only. That tool has reported it correctly.
